static/development/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Forem",
+  "version": "0.1.0",
+  "description": "For Empowering Community",
+  "manifest_version": 2,
+  "permissions": ["storage"],
+  "content_scripts": [
+    {
+      "matches": ["<all_urls>"],
+      "run_at": "document_idle",
+      "js": ["content.js"]
+    }
+  ],
+  "background": { "persistent": true, "scripts": ["hotReload.js"] }
```

Comment:
The main difference between the production and development build is the addition of the hot reload script.